python_modules/libraries/dagster-dbt/dagster_dbt/cloud/resources.py

```diff
@@ -181,6 +181,44 @@ def run_job(self, job_id: int, **kwargs) -> Dict[str, Any]:
         )
         return resp
 
+    def get_runs(
+        self,
+        include_related: Optional[List[str]] = [],
+        job_definition_id: Optional[int] = None,
```

Comment:
job_definition_id should be job_id to align with other methods on this class

Reply:
done